packages/client/get_audio.py

```diff
@@ -29,13 +27,22 @@ def main():
     process_thread.start()
 
     audio = pyaudio.PyAudio()
+    input_device_info = audio.get_device_info_by_index(input_device_index)
+    rate_input = input_device_info.get('defaultSampleRate')
+    if not rate_input:
+        print('Critical: selected input device does not have a sample rate. Exiting')
+        sys.exit(1)
```

Comment:
sys.exit can take a string arg which it will print and then quit with 1. Might be worth using.
https://docs.python.org/3/library/sys.html#sys.exit